crates/router/src/connector/adyen/transformers.rs

```diff
@@ -1112,6 +1112,39 @@ pub enum CardBrand {
     Visa,
     MC,
     Amex,
+    Argencard,
+    Bcmc,
+    Bijcard,
+    Cabal,
+    Cartebancaire,
+    Codensa,
+    Cup,
+    Dankort,
+    Diners,
+    Discover,
+    Electron,
+    Elo,
+    Forbrugsforeningen,
+    Hiper,
+    Hipercard,
+    Jcb,
+    Karenmillen,
+    Laser,
+    Maestro,
+    Maestrouk,
+    Mc,
```

Comment:
We also have `Mc` as an enum variant here. Is that possibly for MasterCard itself?
If yes, can we avoid the duplicates?

Reply:
This is a duplication. Will remove it.